main.cpp

```diff
@@ -19,13 +19,44 @@ using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
 public:
-    CastCallBack(Rewriter& rewriter) {
-        // Your code goes here
-    };
+    CastCallBack(Rewriter& rewriter) : rewriter(rewriter) {}
 
     void run(const MatchFinder::MatchResult &Result) override {
-        // Your code goes here
+		if (const auto *cSCExpr = Result.Nodes.getNodeAs<CStyleCastExpr>("cast")) {
+			const auto& SM = *Result.SourceManager;
+			auto& FM = SM.getFileManager();
+			
+			const auto& LParenLoc = cSCExpr->getLParenLoc();
+			const auto& RParenLoc = cSCExpr->getRParenLoc();
+			
+			// change to static_cast
+			rewriter.RemoveText(LParenLoc, 1);

```

Comment:
You can use `ReplaceText()` instead of `RemoveText()` and `InsertText()`.

Reply:
Yes, I did it in this commit 3642d246f57553f0c475a10bc16ccca5708e26b5